db/migrate/20200417174651_add_unique_index_to_rule_identifiers.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToRuleIdentifiers < ActiveRecord::Migration[5.2]
+  def change
+    add_index(:rule_identifiers, %i[label system rule_id], unique: true)
```

Comment:
This one made me think for a second, what happens for Rules with different SSG, there would be multiple identifiers for those - but there are multiple rule_ids for those anyway so this still holds.